static/js/hotkey.js

```diff
@@ -541,6 +542,8 @@ exports.process_hotkey = function (e, hotkey) {
     switch (event_name) {
         case 'message_actions':
             return popovers.open_message_menu();
+        case 'star_message':
+            return message_flags.toggle_starred_on_selected_message();
```

Comment:
Like @timabbott said, you can just use `return message_flags.toggle_starred(current_msg_list.selected_message())`